packages/labs/gen-wrapper-vue/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "private": true,
+  "name": "@lit-labs/gen-wrapper-vue",
+  "description": "Code generator for Vue wrapper for Lit components",
+  "version": "0.0.1",
+  "author": "Google LLC",
+  "license": "BSD-3-Clause",
+  "bugs": "https://github.com/lit/lit/issues",
+  "type": "module",
+  "main": "lib/index.js",
+  "repository": "lit/lit",
+  "scripts": {
+    "build": "wireit",
+    "test": "wireit",
+    "test:gen": "wireit",
+    "test:update-goldens": "UPDATE_TEST_GOLDENS=true npm run test:gen"
+  },
+  "wireit": {
+    "build": {
+      "command": "tsc --build --pretty",
+      "dependencies": [
+        "../analyzer:build",
+        "../../tests:build",
+        "../vue-utils:build",
+        "../gen-utils:build"
+      ],
+      "files": [
+        "src/**/*",
+        "tsconfig.json"
+      ],
+      "output": [
+        "test-gen",
+        "lib",
+        "index.{js,js.map,d.ts,d.ts.map}",
+        "tsconfig.tsbuildinfo"
+      ],
+      "clean": "if-file-deleted"
+    },
+    "test:gen": {
+      "files": [
+        "goldens/**/*"
+      ],
+      "dependencies": [
+        "build",
+        "../test-projects/test-element-a:build"
```

Comment:
Change this to `:pack` and add a pack command like this: https://github.com/lit/lit/pull/2899/commits/3bc762a91e95f3a6cd53a51f47d4a01bb123adc3

Reply:
Done.